src/processes/follow-topic-process.js

```diff
@@ -61,7 +62,8 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   const channel_info = await anon_to_sign_user(
     serverClient,
     communityMessageFormat.user_id,
-    data.user_id
+    data.user_id,
+    communityMessageFormat.topic_id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `anon_to_sign_user` function now accepts an additional parameter `communityMessageFormat.topic_id`. Ensure that all calls to this function are updated accordingly to prevent potential runtime errors due to missing arguments.
```diff
    data.user_id,
    communityMessageFormat.topic_id
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->